src/components/shared/C3Chart/Colors.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  SoftGrey: '#CCCCCC',
+  LightGrey: '#666666',
+  DarkGrey: '#333333',
+  DarkPurple: '#b58fc1',
```

Comment:
```suggestion
  DarkPurple: '#B58FC1',
```

Reply:
Me parece que prettier en general sugiere lowercase para los colores.

Reply:
Bueno pero pongamos todos los colores iguales en ese caso no? todo mayúscula o todo minúscula.

Reply:
> Bueno pero pongamos todos los colores iguales en ese caso no? todo mayúscula o todo minúscula.

the change is done, everything is in lowercase